src/TPM.jl

```diff
@@ -454,6 +464,23 @@ function export_TPM_results(dirpath, result::SingleTPMResult)
     df = df[:, ["time", "depth", keys_sorted...]]
 
     CSV.write(filepath, df)
+
+    ##= Thermal force on every face of the shape model =##
+    filepath = joinpath(dirpath, "thermal_force.csv")
+
+    nfaces = size(result.face_forces, 1)  # Number of faces of the shape model
```

Comment:
```suggestion
    Ns = size(result.face_forces, 1)  # Number of faces of the shape model
```
This should be better for consistency. Or, replace `Ns` with `nfaces`?

Reply:
Ah, `n_faces` would be the best.

https://github.com/Astroshaper/AsteroidThermoPhysicalModels.jl/pull/116/commits/ea3e9973d31e8094a045e95032107675afc68263

Reply:
```suggestion
    n_faces = size(result.face_forces, 1)  # Number of faces of the shape model
```